<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetAction</class>
 <widget class="QWidget" name="WidgetAction">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>编辑动作</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMinimumSize</enum>
   </property>
   <property name="horizontalSpacing">
    <number>11</number>
   </property>
   <property name="verticalSpacing">
    <number>11</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>动作名称</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>开始帧数</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="startFrame"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>结束帧数</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="endFrame"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>关键点</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="hitPoint"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>startFrame</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
